Libraries/fishhook/fishhook.c

```diff
@@ -103,7 +103,11 @@ static void perform_rebinding_with_section(struct rebindings_entry *rebindings,
               indirect_symbol_bindings[i] != cur->rebindings[j].replacement) {
             *(cur->rebindings[j].replaced) = indirect_symbol_bindings[i];
           }
-          indirect_symbol_bindings[i] = cur->rebindings[j].replacement;
+          if (i < ( sizeof(indirect_symbol_bindings) / sizeof(indirect_symbol_bindings[0]))) {
+            if (cur != NULL) {
```

Comment:
why is this necessary if `cur` is being checked on line 99?

Reply:
Good point, I didn't see that. I'll fix it and test in a little bit

Reply:
Fixed